CONTRIBUTING.md

```diff
@@ -56,7 +56,14 @@ Use [pre-commit](https://pre-commit.com/#installation) to automatically format y
 python3 -m pip install pre-commit  # only required the first time
 pre-commit install
 ```
-That's it! The plugin will run every time you commit any changes. If there are any errors found during the run, fix them and commit those changes. You can even run the plugin manually on all files:
+That's it! The plugin will run every time you commit any changes. If there are any errors found during the run, fix them and commit those changes.
+Pre-commit includes
+[black](https://github.com/psf/black),
+[isort](https://github.com/PyCQA/isort),
+[mypy](http://mypy-lang.org) and
+[flake8](https://flake8.pycqa.org/en/latest/).
```

Comment:
```suggestion
flake8 will also be replaced by [ruff](https://beta.ruff.rs/)
```

Reply:
Updated.

What are your experience with ruff?  Does it replace flake8 completely or flake8 still could be useful?

Reply:
Use ruff… forget flake8.